<launch>
  <node name="aruco_detector" pkg="aruco_detector" type="aruco_detector.py" required="true" output="screen">
  <param name="markerID" type = "int" value="0" />
  <param name="markerLength" type = "double" value="0.2" /> <!--in m-->
  <param name="update_frequency" type = "int" value="100" /> <!--in Hz (max~10)-->
  <param name="cam_topic"  value="/cam_img" /> <!--subscriber topic (camera frame)-->
  <rosparam command="load" file="$(find camera)/config/cam.yaml" />
  </node>
</launch>
